ChangeLog.d/fix_some_resource_leaks.txt

```diff
@@ -0,0 +1,6 @@
+Bugfix
+   * Fixed resource leaks in mbedtls_pk_parse_public_key() in low
+     memory conditions.
+Security
+   * Fix potential memory leak inside mbedtls_ssl_cache_set() with
```

Comment:
(minor) should it be `Fixed` or `Fix`? It should be consistent here (and ideally with the other ChangeLog changes we just made)